### ISSUE TITLE
Service guacspoold.service is failing after sometime

### ISSUE THREAD

**Issue by xpillons** [xpillons/azhop-guac#4]
spooler service is failing after sometime (few hours) and need to be restarted

**Comment by xpillons** [comment]
```
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/cycle/guac/venv/lib/python3.6/site-packages/requests/adapters.py", line 450, in send
    timeout=timeout
  File "/opt/cycle/guac/venv/lib/python3.6/site-packages/urllib3/connectionpool.py", line 727, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/opt/cycle/guac/venv/lib/python3.6/site-packages/urllib3/util/retry.py", line 446, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='ccportal', port=9443): Max retries exceeded with url: /cyclecloud/clusters/remoteviz/status?nodes=true (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fc048b28e10>: Failed to establish a new connection: [Errno -2] Name or service not known',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/cycle/guac/venv/lib/python3.6/site-packages/cyclecloud/session.py", line 39, in _request
    return self._session.request(verb, full_url, params=query, headers=headers, data=body, timeout=self._config["timeout"])
  File "/opt/cycle/guac/venv/lib/python3.6/site-packages/requests/sessions.py", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/cycle/guac/venv/lib/python3.6/site-packages/requests/sessions.py", line 645, in send
    r = adapter.send(request, **kwargs)
  File "/opt/cycle/guac/venv/lib/python3.6/site-packages/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='ccportal', port=9443): Max retries exceeded with url: /cyclecloud/clusters/remoteviz/status?nodes=true (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fc048b28e10>: Failed to establish a new connection: [Errno -2] Name or service not known',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/cycle/guac/venv/lib/python3.6/site-packages/guac/daemon.py", line 68, in main
    autoscale_guac(config, ctx_handler=ctx_handler)
  File "/opt/cycle/guac/venv/lib/python3.6/site-packages/guac/autoscaler.py", line 58, in autoscale_guac
    demand_calculator = calculate_demand(config, guac_env, guac_driver, ctx_handler, node_history)
  File "/opt/cycle/guac/venv/lib/python3.6/site-packages/guac/autoscaler.py", line 197, in calculate_demand
    config, guac_env, guac_driver, ctx_handler, node_history
  File "/opt/cycle/guac/venv/lib/python3.6/site-packages/guac/autoscaler.py", line 159, in new_demand_calculator
    node_mgr = new_node_manager(config, existing_nodes=guac_env.scheduler_nodes)
  File "/opt/cycle/guac/venv/lib/python3.6/site-packages/hpc/autoscale/hpclogging.py", line 199, in apitrace_wrapper
    ret_val = function(*args, **kwargs)
  File "/opt/cycle/guac/venv/lib/python3.6/site-packages/hpc/autoscale/node/nodemanager.py", line 1200, in new_node_manager
    ret = _new_node_manager_79(new_cluster_bindings(config), config)
  File "/opt/cycle/guac/venv/lib/python3.6/site-packages/hpc/autoscale/node/nodemanager.py", line 1265, in _new_node_manager_79
    cluster_status = cluster_bindings.get_cluster_status(nodes=True)
  File "/opt/cycle/guac/venv/lib/python3.6/site-packages/hpc/autoscale/ccbindings/legacy.py", line 183, in get_cluster_status
    self.session, self.cluster_name, nodes
  File "/opt/cycle/guac/venv/lib/python3.6/site-packages/cyclecloud/api/clusters.py", line 378, in get_cluster_status
    _status, _response = session.request(_request_context, 'GET', query=_query, headers=_headers, body=_body, expected_responses=_responses)
  File "/opt/cycle/guac/venv/lib/python3.6/site-packages/cyclecloud/session.py", line 50, in request
    r = self._request(verb, full_url, query, headers, body)
  File "/opt/cycle/guac/venv/lib/python3.6/site-packages/cyclecloud/session.py", line 41, in _request
    raise Exception("Unable to connect to %s - %s" % (self._config["url"], e))
Exception: Unable to connect to https://ccportal:9443/cyclecloud - HTTPSConnectionPool(host='ccportal', port=9443): Max retries exceeded with url: /cyclecloud/clusters/remoteviz/status?nodes=true (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fc048b28e10>: Failed to establish a new connection: [Errno -2] Name or service not known',))
```

**Comment by xpillons** [comment]
fixed in 0.1.13
